webknossos-datastore/app/com/scalableminds/webknossos/datastore/models/DataRequests.scala

```diff
@@ -50,7 +50,7 @@ case class WebknossosAdHocMeshRequest(
     mag: Vec3Int,
     cubeSize: Vec3Int, // In target mag
     segmentId: Long,
-    scale: Vec3Double,
+    scale: Vec3Double, // voxel size factor, assumed to be in dataset’s unit
```

Comment:
Then maybe call the member `voxeSizeFactorInUnit`?  That's at least the naming schema which I used in the frontend

Reply:
Ok, I did that now in the backend. Could you adapt the frontend accordingly?